crates/storage/trie/db/in_memory.rs

```
@@ -25,4 +25,14 @@ impl TrieDB for InMemoryTrieDB {
         self.inner.lock().unwrap().insert(key, value);
         Ok(())
     }
+
+    fn put_batch(&self, key_values: Vec<(Vec<u8>, Vec<u8>)>) -> Result<(), TrieError> {
+        let mut db = self.inner.lock().unwrap();
```

Unwrap here, let's return an error.

---

Removed unwraps here [4845ce8](https://github.com/lambdaclass/ethrex/pull/1358/commits/4845ce80da201a8ede0512b33fb76d6459e351a9)